styles/variables-silicon.less

```diff
@@ -10,12 +10,19 @@
 
 // Button
 // ---------------------------------------
-@agate-button-border-radius: 6px;
+@agate-button-border-radius: 37px;
```

Comment:
Does the border-radius really need to be exactly 37 pixels? Could this instead be a simpler or more straightforward value, like:
```css
@agate-button-border-radius: @agate-button-height;
```
or if you want to get really precise:
```css
@agate-button-border-radius: (@agate-button-height / 2);
```
`border-radius` value really only matters up to half of the height of the element when all corners are specified. So any value is ok; so we might as well set one that is easy for the browser to calculate.